packages/components/src/Tabs/Tabs.tsx

```diff
@@ -72,13 +86,20 @@ export function Tabs({
   return (
     <div className={styles.tabs}>
       <div className={overflowClassNames}>
-        <ul role="tablist" className={styles.tabRow} ref={tabRow}>
+        <ul
+          role="tablist"
+          className={styles.tabRow}
+          ref={tabRow}
+          onKeyDown={handleKeyDown}
+        >
           {React.Children.map(children, (tab, index) => (
             <InternalTab
               label={tab.props.label}
               selected={activeTab === index}
               activateTab={activateTab(index)}
               onClick={tab.props.onClick}
+              ref={el => (tabRefs.current[index] = el)}
+              tabIndex={activeTab === index ? 0 : -1}
```

Comment:
This ensures that only the active tab is focusable using the tab key.  So if you have two tabs, and the first tab has a focusable child, when focus is on the tab and you hit tab key, the focus will go to the child and not the next tab.
Instead, arrow keys will help you continue to navigate to the next tab.